daq/host.py

```diff
@@ -174,6 +176,12 @@ def _get_static_ip(self):
     def _get_dhcp_mode(self):
         return self._loaded_config['modules'].get('ipaddr', {}).get('dhcp_mode')
 
+    def _get_unique_upload_path(self, file_name):
+        basename = os.path.basename(file_name)
+        unique_name = "%s-%s.%s" % (self.target_port, self.test_name, basename) \
```

Comment:
use intermediate variable rather than line break... same number of lines but easier to parse